### PR TITLE
list of supported currency added by coindesk

### PR DESCRIPTION
**Pull request by saravanan10393**
Reference issue #3 

Add the option to choose between various currency formats supported by coindesk.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -6,16 +6,23 @@ import moment from 'moment';
...
+      selectedCurrency: 'USD',
+      supportedCurrencies: [
+        {"currency":"USD","country":"United States Dollar"},
+      ],
       data: null,
       hover: null,
       activePoint: null
     }
+
+    this.changeCurrency = this.changeCurrency.bind(this);
```
[the remark this answers is not in the document]
Comment (saravanan10393) [reply src/App.js]: If no bind means then need to use inline function to avoid using bind.

[the remark this answers is not in the document]
Comment (benhalverson) [reply src/App.js]: an inline function is ok with me

[the remark this answers is not in the document]
Comment (benhalverson) [reply src/App.js]: also check out this article https://medium.freecodecamp.org/react-binding-patterns-5-approaches-for-handling-this-92c651b5af56

[the remark this answers is not in the document]
Comment (saravanan10393) [reply src/App.js]: Ok @benhalverson , I will check it out.

[the remark this answers is not in the document]
Comment (benhalverson) [reply src/App.js]: @saravanan10393 are you still working on this?

**Pull request closed without merging**